tsl/test/expected/cagg_ddl-14.out

```diff
@@ -2116,3 +2116,20 @@ SELECT * FROM _timescaledb_catalog.compression_settings;
 -------+-----------+---------+--------------+--------------------
 (0 rows)
 
+-- hash partitioning test
+CREATE TABLE test_by_hash (id BIGINT, value FLOAT8);
+SELECT create_hypertable('test_by_hash', by_hash('id', 8));
```

Comment:
@fabriziomello with my PR https://github.com/timescale/timescaledb/pull/6993 we won't allow creation of hypertables with `by_hash`

Reply:
Fixed!